src/components/chart/chart.js

```diff
@@ -137,6 +153,10 @@ class HighchartsBaseChart {
             // Merge the bar chart options with the existing config
             this.config = this.mergeConfigs(this.config, barChartOptions);
         }
+        if (this.chartType === 'columnrange') {
+            // Merge the bar chart options with the existing config
+            this.config = this.mergeConfigs(this.config, columnRangeChartOptions);
```

Comment:
Heads up that in my reference line pr I moved `mergeConfigs` to a utilities file - we'll just need to make sure this line, and the similar ones below, change to `mergeConfigs` (without the `this.`) when we merge them.